PKHeX/MainWindow/Main.cs

```diff
@@ -3279,6 +3282,9 @@ private void clickSet(object sender, EventArgs e)
         private void clickDelete(object sender, EventArgs e)
         {
             int slot = getSlot(sender);
+            if ((SAV.Version == GameVersion.SN || SAV.Version == GameVersion.MN) && IsLinkedSlot(slot, CB_BoxSelect.SelectedIndex)) {
```

Comment:
I think it would increase maintainability to move `(SAV.Version == GameVersion.SN || SAV.Version == GameVersion.MN)` to IsLinkedSlot, having the function return false if the version is not sun or moon.  The same applies to the 3 other uses in the same file.